### PR TITLE
Contact page

### PR DESCRIPTION
**Pull request by BuzzDeejayLord**
I had added the contact page that should be able to send an e-mail to hestiasupport@gmail.com. Also I have added the presentation and my time tracking as i didn't do it in the last two iterations

### CODE REVIEW

**Comment by BuzzDeejayLord** [comment]
correction... hestia.contact.email@gmail.com

File: Hestia/Hestia/UITa.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+using Foundation;
```
Comment (MarcF12) [inline Hestia/Hestia/UITa.cs]: I am wondering what this class is used for?

File: Hestia/Hestia/frontend/SettingsScreen/UITableViewControllerContacts.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        MFMailComposeViewController mailController;
+
+        public UITableViewControllerContacts(IntPtr handle) : base(handle)
+        {
+        }
+
+        //Opinion
+        partial void UIButton105005_TouchUpInside(UIButton sender)
+        {
+            if(MFMailComposeViewController.CanSendMail){
```
Comment (MarcF12) [inline Hestia/Hestia/frontend/SettingsScreen/UITableViewControllerContacts.cs]: There is a lot of duplicated code in these three methods. You should extract a methods from this. Also, "this is a test" should still be changed to something more informative I guess?

**Comment by BuzzDeejayLord** [comment]
Look on slack

**Comment by MarcF12** [comment]
See the commit I pushed for what I had in mind. Please let me know if you agree then I will merge.

**Comment by MarcF12** [comment]
It seems to work.

**Pull request merged**